tools/runtest

```diff
@@ -662,7 +662,7 @@ if [ "$serial" = 'yes' -o "$mpi" = 'yes' -o "$cuda" = 'yes' -o "$cudampi" = 'yes
       echo  "Error: $qbindir/quick not found."
       exit 1
     fi
-    if [ ! -x "$testdir/test-api" ] | [ ! -x "$qbindir/test-api" ]; then
```

Comment:
This works but why do you need the round brackets? I always have to look up these things for bash and/or sh but it works for both without the round brackets:
```
sh-3.2$ touch x1 x2
sh-3.2$ chmod u+x x2
sh-3.2$ if ! [ -x x1  -o -x x2 ]; then echo "not executable"; fi
sh-3.2$ if ! [ -x x1  -o -x x3 ]; then echo "not executable"; fi
not executable
sh-3.2$ 
```